src/components/NoteInput/NoteInput.scss

```
@@ -67,6 +67,16 @@ $note-input__input-right: 40px;
   transition: all 0.08s ease-out;
 }
 
+.toast-button {
+  background-color: $color-backlog-blue;
+  border-style: none;
+  border-radius: 12px;
```

I'd prefer to use 8px for radius, due to the SettingsButtons also have a 8px radius, so its consistent.

```suggestion
  border-radius: 8px;
```